package.json

```diff
@@ -8,6 +8,7 @@
     "test": "NODE_OPTIONS=--experimental-vm-modules npx mocha --timeout 10000 -r dotenv/config dotenv_config_path=src/test-fixtures/.env.testing src/app.test.js "
   },
   "dependencies": {
+    "@digitalcredentials/credential-status-manager-db": "github:digitalcredentials/credential-status-manager-db#initial-implementation",
```

Comment:
We will update this dependency after we deploy `@digitalcredentials/credential-status-manager-db` to npm.